### PR TITLE
Bugfix/general bug review

### PR DESCRIPTION
**Pull request by martinagrzybowska**
[hooks]
- client: moving to layout for taking too long in canister. When navigating to homepage or sign in page no need to login

[logout]
- ensuring server cookies and context is deleted on logout
- solving logout issue in canister by creating separate logout action page

[profile]
- introducing a delay in fetching followers and followings, due to race conditions with user posts fetch

[misc]
- solving smooth scroll problems in homepage
- homepage now in forever dark mode
- disabling bookmarks and notifications fo all users in navigation
- adding Toaster component, to be used in situations where a click on an element is to trigger a Toast
- just shooting a solution for identity window appearing in popup by adding windowOpenerFeatures (might not work)
- live feed now fetches only last 24 hours to the past (huge overhead if left unchecked)
- default page after signup is now feed

[post]
- disabling bookmarks
- share button now copies user url
- adding additional support for finding and parsing deprecated nip-10 replies, making the experience of fetching parent thread, as well as event replies, a little bit better
- localizing cache options

[sign-in]
- fixing buttons not firing click events out of nowhere
- fixing problems with sign-out and subsequent sign-in of a different user, where the previous user was still retained in feed

### CODE REVIEW

File: src/frontend/src/lib/components/modals/create-post/CreatePostModal.svelte [hunk 1]
```diff
@@ -92,7 +91,7 @@
 {#if $modalStore[0]}
   <div class="card p-4 w-modal shadow-xl space-y-4">
     <header class="text-2xl font-bold">{$modalStore[0].title}</header>
-    <article>{$modalStore[0].body}</article>
+    <!--    <article>{$modalStore[0].body}</article>-->
```
Comment (viktorkovarik) [inline src/frontend/src/lib/components/modals/create-post/CreatePostModal.svelte:94]: Can we remove this or do we have to keep there this commented out code?

**Review by viktorkovarik — APPROVED** [review]
LGTM, only one thing, there is a lot of commented out code. Do we want keep it there (and then put it back in the future) or do we want to remove it completely?

File: src/frontend/src/lib/components/modals/create-post/CreatePostModal.svelte (hunk 1 above)
Comment (martinagrzybowska) [reply src/frontend/src/lib/components/modals/create-post/CreatePostModal.svelte:94]: Need to keep it there for now

Comment (viktorkovarik) [reply src/frontend/src/lib/components/modals/create-post/CreatePostModal.svelte:94]: Alright then! I've tested it locally, feel free to merge it. Then we will check how it behaves on dev environment.

**Pull request merged**